src/backend/src/services/change-request.services.ts

```diff
@@ -91,11 +92,13 @@ export default class ChangeRequestsService {
       // else if cr is for a wp: update the budget and duration based off of the proposed solution
       if (!foundCR.wbsElement.workPackage && foundCR.wbsElement.project) {
         const newBudget = foundCR.wbsElement.project.budget + foundPs.budgetImpact;
-        const change = {
-          changeRequestId: crId,
-          implementerId: reviewer.userId,
-          detail: buildChangeDetail('Budget', String(foundCR.wbsElement.project.budget), String(newBudget))
-        };
+        const change = createChange(
+          'Budget',
+          String(foundCR.wbsElement.project.budget),
```

Comment:
we shouldnt cast to a string, that defeats the purpose of the function taking in any.